tests/src/test/java/alluxio/client/FileOutStreamIntegrationTest.java

```diff
@@ -96,11 +97,14 @@ private void writeTest3Util(AlluxioURI filePath, int len, CreateFileOptions op)
   public void writeSpecifyLocalTest() throws Exception {
     AlluxioURI filePath = new AlluxioURI(PathUtils.uniqPath());
     final int length = 2;
-    FileOutStream os = mFileSystem.createFile(filePath, mWriteLocal);
+    FileOutStream os = mFileSystem.createFile(filePath,
+        CreateFileOptions.defaults().setWriteType(WriteType.CACHE_THROUGH).
+            setLocationPolicy(new LocalFirstPolicy()));
     os.write((byte) 0);
     os.write((byte) 1);
     os.close();
-    checkWrite(filePath, mWriteLocal.getUnderStorageType(), length, length);
+    checkWrite(filePath, CreateFileOptions.defaults().setWriteType(WriteType.CACHE_THROUGH).
```

Comment:
`CreateFileOptions.defaults().setWriteType(WriteType. CACHE_THROUGH).getUnderStorageType()` is equivalent to `UnderStorageType.SYNC_PERSIST`